### PR TITLE
Mm 52259 system message ignore colorize

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR sets the default value for the username color using the theme provided value replacing the hardcoded values (#000000) which was causing the system messages not displaying correctly in dark themes.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-52259

#### Screenshots
Before:
*Dark theme:
Screenshot 2023-06-05 at 16 35 18

*Light Theme:
Screenshot 2023-06-05 at 16 19 15

After:
Screenshot 2023-06-05 at 16 33 08
Screenshot 2023-06-05 at 16 33 52

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/user_profile/user_profile.tsx [hunk 1]
```diff
@@ -92,7 +90,7 @@ export default class UserProfile extends PureComponent<UserProfileProps> {
 
         const ariaName: string = typeof name === 'string' ? name.toLowerCase() : '';
 
-        let userColor = '#000000';
+        let userColor = theme?.centerChannelColor;
```
Comment (pvev) [inline webapp/channels/src/components/user_profile/user_profile.tsx:93]: Note this is the real change, the remaining stuff in this PR is code clean up, the removal unused components props

**Review by JulienTant — APPROVED** [review]
thanks @pvev

**Comment by Willyfrog** [comment]
@pvev should this have QA approval or the QA deferred label?

**Comment by pvev** [comment]
> @pvev should this have QA review or the QA deferred label?

this one can be QA deferred, small change with very low risk of regressions.

**Pull request merged**